code/game/gamemodes/devil/devilinfo.dm

```diff
@@ -138,6 +139,13 @@ var/global/list/lawlorify = list (
 	return pick(BANISH_WATER, BANISH_COFFIN, BANISH_FORMALDYHIDE, BANISH_RUNES, BANISH_CANDLES, BANISH_DESTRUCTION, BANISH_FUNERAL_GARB)
 
 /datum/devilinfo/proc/add_soul(datum/mind/soul)
+
+	var/obj/screen/devil/soul_counter/K = new /obj/screen/devil/soul_counter
```

Comment:
Can you please explain me, because the code is not ready, and thats where it's going wrong